src/pages/Admin/Charity/Banking/RecipientDetails/types.ts

```diff
@@ -1,7 +1,7 @@
 import { CreateRecipientRequest } from "types/aws";
-import { Country } from "types/countries";
-import { OptionType } from "types/utils";
-import { Asset } from "components/registration";
+import { OptionType } from "types/components";
+import { Country } from "types/components";
+import { Asset } from "components/FileDropzone";
```

Comment:
moved Filedropzone from `components/registration` to `components/` as it is used also in admin now